bin/npmvsyarn.js

```diff
@@ -9,56 +9,73 @@ let prettyMs = require('pretty-ms'),
 
 let packageName = process.argv[2];
 
-// Check if yarn exists, if not install it.
```

Comment:
@siddharthkp you have accidentally removed this logic which checks if yarn already exists. If not, installs it.